### ISSUE TITLE
Screen Reader doesn't announce position for group Headers which is present inside 'Sample queries' tab item.

### ISSUE THREAD

**Issue by ElinorW** [microsoftgraph/microsoft-graph-explorer-v4#1149]
**Test Environment:**
OS: Windows 11 Version 21H2 (OS Build 22000.100)
Microsoft Edge Dev Build 93.0.957.0
URL: Graph Explorer - Microsoft Graph
Screen Reader: Narrator

**Repro Steps:**
Open each given URL in Microsoft Edge Dev Browser.
Open Narrator using 'Win+ Ctrl + Enter' key.
Navigate to the 'Sample queries' tab item in the left side navigation landmark and select it.
Navigate to 'Getting started' Button in the menu-list using tab key.
Observe the issue.

**Actual Result:**
Screen Reader doesn't announce position for group Headers which is present inside 'Sample queries' tab item. For Getting Started group header Screen Reader is announcing 'Getting Started has 6 result Group'.

**Expected Result:**
Screen Reader should announce position for group Headers which is present inside 'Sample queries' tab item. For Getting Started group header Screen Reader should announce 'Getting Started has 6 results 1 of 23 Level 1 Collapsed'.

Suggestive Fix: add role='treeitem' with every group header div.

**Observation:**
I have seen "aria-setsize" and "aria-posinset'' attribute are present in the code inside Getting Started div which is basically used to covey the position of an element but it didn't conveyed to Screen Reader due to absence of role attribute in that same div.

Note: Issue repro with NVDA screen reader also.

**User Impact:**
Screen Reader user are impacted here as when they navigate to 'Get Started(8)' using tab key Screen Reader is announcing 'Getting Started has 6 items Group' so clearly NO hint for Screen Reader user that there are other element below it, that he/she can access with arrow keys because when he/she press tab key again its jump to outside element 'Get'.
